test/e2e/specs/site-editor/command-center.spec.js

```diff
@@ -17,34 +17,29 @@ test.describe( 'Site editor command center', () => {
 		await admin.visitSiteEditor();
 	} );
 
-	test.skip( 'Open the command center and navigate to the page create page', async ( {
+	test( 'Open the command center and navigate to the page create page', async ( {
 		page,
 	} ) => {
+		await page.focus( 'role=button[name="Open command center"i]' );
 		await page.keyboard.press( 'Meta+k' );
+		await page.keyboard.type( 'new page' );
 		const newPageButton = page.locator(
-			'role=option[name="Create a new page"i]'
+			'role=option[name="Add new page"i]'
 		);
 		await expect( newPageButton ).toBeVisible();
-
-		// Type a random post title
-		await page.keyboard.type( 'E2E Test Post' );
-		await page.click(
-			'role=option[name="Create a new post \\"E2E Test Post\\""i]'
-		);
+		await newPageButton.click();
 
 		await page.waitForSelector( 'iframe[name="editor-canvas"]' );
 		const frame = page.frame( 'editor-canvas' );
-		const postTitleInput = frame.locator(
-			'role=textbox[name=/Add title/i]'
-		);
-		await expect( postTitleInput ).toHaveText( 'E2E Test Post' );
+		await expect(
+			frame.locator( 'role=textbox[name=/Add title/i]' )
+		).toBeVisible();
 	} );
 
-	test.skip( 'Open the command center and navigate to a template', async ( {
+	test( 'Open the command center and navigate to a template', async ( {
 		page,
 	} ) => {
-		await page.keyboard.press( 'Meta+k' );
-
+		await page.click( 'role=button[name="Open command center"i]' );
```

Comment:
@kevin940726 suggests(as nit though) the `getByRole` usage(see similar [comment](https://github.com/WordPress/gutenberg/pull/50797#discussion_r1200236185))